cmd/main.go

```diff
@@ -101,6 +102,8 @@ func main() {
 	flag.DurationVar(&cacheCleanupInterval, "cache-cleanup-interval", time.Minute,
 		"The interval at which expired cache entries are removed. "+
 			"A shorter interval ensures frequent cleanup but may impact performance.")
+	flag.StringVar(&kubestaticID, "kubestatic-id", "", "A unique id kubestatic uses to tag its resources on AWS. "+
+		"The environment name could be used here. Has to be set")
```

Comment:
Improvement suggestion
```suggestion
	flag.StringVar(&kubestaticID, "kubestatic-id", "", "A required unique identifier that Kubestatic uses to tag and track ownership of its AWS resources.")
```